### ISSUE TITLE
Any chance to get this integrated into Matplotlib?

### ISSUE THREAD

**Issue by claell** [riley-x/SankeyFlow#4]
As the current Sankey Matplotlib package seems to be designed for a different purpose, adding this code would be super helpful and ensuring continuous maintaining.

**Comment by riley-x** [comment]
I hadn't really thought about that, but I can def look into it, though I won't be able to commit much time at the moment.

**Comment by claell** [comment]
Alright, thanks for your interest in that! Let's see, maybe we can see this in upstream, one day.

**Comment by hyvenglaven** [comment]
I agree this would be a wonderful addition to `matplotlib`! I think the community would also support such an integration. One issue the community would run into in trying to incorporate this into `matplotlib` is `SankeyFlow` is under a GPL license and `matplotlib` uses the more developer-friendly BSD style Python Software Foundation License (PSFL). @riley-x would you consider changing the license of SankeyFlow to a BSD style license to enable developers to incorporate your code into `matplotlib` under the same license terms, and, in the meantime, to use it stand-alone alongside `matplotlib` without the extra restrictions? This would likely help it grow its user-base and make it more likely the community would support integration efforts.

Thanks for putting this together--I really like the straightforward interface and `matplotlib` backing!

**Comment by riley-x** [comment]
Sure I switched to BSD-2, happy for someone to help merge it into MPL.

**Comment by hyvenglaven** [comment]
@riley-x thanks! I'll definitely use it in my projects now and see if I can find time to help merge into `matplotlib`. Would you mind tagging a new release on github/pypi so that the new license is associated with a specific version and shows up on pypi?
